plugins/rapid7_insightvm/plugin.spec.yaml

```diff
@@ -1920,17 +1920,26 @@ actions:
         type: "[]string"
         example: ["192.0.2.3", "192.0.2.10-192.0.2.20", "ADSRV.local"]
         required: false
+      override_blackout:
+        title: Override Blackout
+        description: Whether to request for the override of an scan blackout window
```

Comment:
Suggested reword: 'Set True to override any scan blackout window'

Reply:
Done